### ISSUE TITLE
Override ToString for toString in C#?

### ISSUE THREAD

**Issue by tjpalmer** [temperlang/temper#65]
If we define a `toString` method in Temper, should we add `override` or `new` in C# for [ToString](https://learn.microsoft.com/en-us/dotnet/api/system.object.tostring?view=net-8.0). Which is a better idea? Any other methods to watch for?

**Comment by mikesamuel** [comment]
Yeah, currently our be-csharp translations generate warnings like:

> warning CS0114: 'SomeType.ToString()' hides inherited member 'object.ToString()'. To make the current member override that implementation, add the `override` keyword. Otherwise add the new keyword. [.../src/SecureComposition.csproj]

I think we probably want override.  Once we have meta interfaces and something supports stringification, this will probably become easier and `new` would be the preferred choice for things that we do not intend as overrides.
